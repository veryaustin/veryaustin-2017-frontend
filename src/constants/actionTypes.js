export const DATA_FETCH = 'DATA_FETCH';