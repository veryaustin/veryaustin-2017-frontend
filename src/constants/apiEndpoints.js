export const DATA_ROOT_URL = "https://veryaustin-55602.firebaseio.com";
export const STATIC_ASSETS_URL = "http://static.veryaustin.com";
