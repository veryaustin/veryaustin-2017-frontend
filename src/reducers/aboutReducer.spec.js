import * as types from '../constants/actionTypes';
import reducer from './aboutReducer';

describe('About Reducer',() => {
  it('should return the initial state', () => {
    const action = {type: 'unknown'};
    expect(reducer(undefined, action)).toMatchSnapshot();
  });

  it('should handle the ABOUT_FETCH action', () => {
    const initialState = {};

    const testData = {
      title: 'Test Title',
      intro: 'This is a test of the about introduction',
      photo: 'http://test_about_photo_url'
    };

    const action = {
     type: types.ABOUT_FETCH,
      payload: {data: testData}
    };
    
    expect(reducer(initialState, action)).toMatchSnapshot();
  });
});