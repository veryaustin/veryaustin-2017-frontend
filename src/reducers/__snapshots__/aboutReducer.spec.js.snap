exports[`About Reducer should handle the ABOUT_FETCH action 1`] = `Object {}`;

exports[`About Reducer should return the initial state 1`] = `Object {}`;
